programs/openbook-v2/src/instructions/place_order.rs

```diff
@@ -71,19 +74,8 @@ pub fn place_order(ctx: Context<PlaceOrder>, order: Order, limit: u8) -> Result<
         Side::Bid => {
             let free_quote = position.quote_free_native;
 
-            let max_quote_including_fees = if let Some(order_id) = order_id {
-                let price = match order.params {
-                    OrderParams::OraclePegged { peg_limit, .. } => peg_limit,
```

Comment:
how is this case covered now?

Reply:
`placed_quantity * market.quote_lot_size * price` is the `posted_quote_native` returned, which together with `maker_fess`, will be zero in case the order is not posted, so no need to branch here